src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader/ReaderMetricsProcessor.java

```diff
@@ -754,6 +828,28 @@ public void emitNodeMetrics(long currWindowStartTime, MetricsDB metricsDB) throw
     }
   }
 
+  @VisibleForTesting
+  public void readBatchMetricsEnabledFromConf() {
+    Path filePath = Paths.get(Util.DATA_DIR, BATCH_METRICS_ENABLED_CONF_FILE);
+
+    Util.invokePrivileged(
+            () -> {
+              try (Scanner sc = new Scanner(filePath)) {
+                String nextLine = sc.nextLine();
+                boolean oldValue = batchMetricsEnabled;
+                boolean newValue = Boolean.parseBoolean(nextLine);
+                if (oldValue != newValue) {
+                  batchMetricsEnabled = newValue;
+                  LOG.info("Batch metrics enabled changed from {} to {}", oldValue, newValue);
+                }
+              } catch (IOException e) {
+                LOG.error("Error reading file '{}': {}", filePath.toString(), e);
+                e.printStackTrace();
```

Comment:
We should keep this as-is to stay consistent with readRcaEnabledFromConf.

Reply:
We may not want to do everything for consistency as much as we want to do the right thing. :) This may be redundant as the stack will already be captured in the logs.

Reply:
Sure, will just remove it.